ansible/roles/tileserver/tasks/main.yml

```diff
@@ -96,7 +96,7 @@
       owner: postgres
       group: postgres
     notify:
-      - systemctl reload postgresql
+      - systemd reload postgresql
```

Comment:
my systemd has no reload, are you sure about this?

Reply:
The handler is defined in line 200 of that file, it just needs to be the same.